test/Ported/Issues/RavenDB_15497.ts

```diff
@@ -54,9 +54,9 @@ describe("RavenDB_15497", function () {
 
             await assertThrows(() => session.saveChanges(), e => {
                 assertThat(e.name)
-                    .isEqualTo("TimeoutException");
```

Comment:
That isn't that easy. There's some logic based on that somewhere. I think it's better leave it for Marcin to handle.